src/views/certifications/components/gradient-card/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode } from 'react'
+import classNames from 'classnames'
+import { ProgramSlug } from 'views/certifications/types'
+import s from './gradient-card.module.css'
+
+/**
+ * Renders a card with a pretty gradient drop shadow.
+ *
+ * Note: in the future we want to enhance this component by adding a
+ * gradient border style as well. For now, we have the drop shadow only.
+ */
+function GradientCard({
```

Comment:
💖 it looks nice 💅🏻

Reply:
Not sure what the gradient border looks like, but here is an implementation to add gradient borders that might be helpful to pull ideas from. https://joebell.co.uk/sketches/loading-disco